prow/git/git.go

```diff
@@ -32,7 +32,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 
-	prowgithub "k8s.io/test-infra/prow/github"
+	git "k8s.io/test-infra/prow/git/types"
```

Comment:
since `k8s.io/test-infra/prow/git` is also a package, it's a bit confusing to name this import as `git`. Why not use `types` just like in any other file? If there is any conflict, I would expect this to be named `gittypes`

Reply:
Done